indico/modules/categories/controllers/management.py

```diff
@@ -34,3 +35,14 @@ def _category_query_options(self):
     def _process(self):
         return WPCategoryManagement.render_template('management/content.html', self.category, 'content',
                                                     categories=self.category.children)
+
+
+class RHSortSubcategories(RHManageCategoryBase):
+    def _checkParams(self):
+        RHManageCategoryBase._checkParams(self)
+        self.subcategory_ids = request.json['categories']
```

Comment:
I'd just do that in `_process` - no need to put all request data on `self` unless it's useful for inheritance etc